neqo-qpack/src/stats.rs

```diff
@@ -0,0 +1,13 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// Tracking of some useful statistics.
+
+#[derive(Default, Debug)]
+/// `QPack` statistics
+pub struct Stats {
+    pub dynamic_table_used: bool,
```

Comment:
Why not count the number of entries inserted instead?

One problem with this as it is is that you count a change to the maximum capacity as "use", which isn't really what you want.  You want the number of insertions AND the number of references to the table to be non-zero.

Reply:
I actually did not want to merge this as it is. I wrote this quickly for the interop. 
Yes all your comments are valid and should be change.

We need some stats like this anyway for telemetry (I am working on a doc that collects all probes we want.)